log/syslog_formatter.go

```diff
@@ -0,0 +1,81 @@
+package log
+
+import (
+	"fmt"
+	"log/syslog"
+	"os"
+
+	"github.com/Sirupsen/logrus"
+)
+
+var ceeTag = []byte("@cee:")
+
+type syslogger struct {
+	wrap logrus.Formatter
+	out  *syslog.Writer
+}
+
+func newSyslogger(appname string, level string, fmter logrus.Formatter) (*syslogger, error) {
+	priority := getLocal(level)
+	out, err := syslog.New(priority, appname)
+	return &syslogger{
+		out:  out,
+		wrap: fmter,
+	}, err
+}
+
+func getLocal(level string) syslog.Priority {
```

Comment:
Well, it returns a `syslog.Priority`. The stdlib docs say:

"The Priority is a combination of the syslog facility and severity."

So I guess priority would actually be correct here...